meta-mender-core/recipes-bsp/grub-mender-grubenv/grub-mender-grubenv_git.bb

```diff
@@ -6,4 +6,4 @@ SRCREV = "${GRUB_MENDER_GRUBENV_REV}"
 PV = "1.3.0+git${SRCREV}"
 
 LICENSE = "Apache-2.0"
-LIC_FILES_CHKSUM = "file://LICENSE;md5=b4b4cfdaea6d61aa5793b92efd42e081"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=30b4554c64108561c0cb1c57e8a044f0"
```

Comment:
This one shouldn't be updated unless you also update `GRUB_MENDER_GRUBENV_REV` (which isn't necessary unless something else has changed too, IMHO).